CapstoneProject/Models/MoodDetection.m

```diff
@@ -23,54 +25,65 @@ - (instancetype)init {
         NSArray *angryKeyWords = @[@"angry", @"mad", @"upset"];
         
         
-        self.stopWordsForMoods =  @{sadKeyWords: @"sad", boredKeyWords: @"bored", happyKeyWords: @"happy", angryKeyWords: @"angry"};
+        self.stopWordsForMoods =  @{@"sad": sadKeyWords, @"bored": boredKeyWords, @"happy": happyKeyWords, @"angry": angryKeyWords};
         
         self.moodCount = [NSMutableDictionary dictionaryWithObjectsAndKeys:
-                          @0, @"sad",
-                          @0, @"bored",
-                          @0, @"happy",
-                          @0, @"angry",
+                          @"sad", @0,
+                          @"bored", @0,
+                          @"happy", @0,
+                          @"angry", @0,
                           nil];
```

Comment:
i think the signature of  dictionaryWithObjectsAndKeys looks like this 
`NSDictionary *dict = [NSDictionary dictionaryWithObjectsAndKeys:
    @"value1", @"key1", @"value2", @"key2", nil];
` https://developer.apple.com/documentation/foundation/nsdictionary/1574181-dictionarywithobjectsandkeys

its working because whenever you call intValue on nil, it returns 0, so for [_moodCount[@"angry"] intValue] if you dont have @"angry" key in your dictionary, it would return 0. Technically here you could create an empty dictionary, and it would work. Let me know if you have questions regarding to this.



i think for stopWordsForMoods now it looks correct, cause it should be 
NSDictionary *dictionaryLiteral = @{@"key1": @"value1", @"key2": @"value2"};